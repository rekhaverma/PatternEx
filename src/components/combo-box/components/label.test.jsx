import React from 'react';
import OptionLabel from './label.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<OptionLabel />', () => {
  it('should match with snapshot with "hasCancel"', () => {
    const props = {
      hasCancel: true,
    };
    const wrapper = shallow(<OptionLabel {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match with snapshot without "hasCancel"', () => {
    const wrapper = shallow(<OptionLabel />);
    expect(wrapper).toMatchSnapshot();
  });
});
