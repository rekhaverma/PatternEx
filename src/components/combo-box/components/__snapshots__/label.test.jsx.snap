// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OptionLabel /> should match with snapshot with "hasCancel" 1`] = `
<div
  className="comboBox__label"
>
  <span>
    Untitled label
  </span>
  <span
    className="icon-close"
    onClick={[Function]}
  />
</div>
`;

exports[`<OptionLabel /> should match with snapshot without "hasCancel" 1`] = `
<div
  className="comboBox__label"
>
  <span>
    Untitled label
  </span>
</div>
`;
